catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/impl/CatalogFrameworkImplTest.java

```diff
@@ -1659,6 +1659,8 @@ public void testGetResourceFromCache() throws Exception {
         ResourceResponse response = framework.getResource(request, federatedSite1Name);
 
         assertThat(response, is(ResourceResponse.class));
+        Metacard responseMetacard = (Metacard) response.getProperties().get("metacard");
+        assertThat(responseMetacard, is(Metacard.class));
```

Comment:
I assume this was done prior to is() being deprecated.  Updating the tests to use the newer call should be part of a tech-debt story. For now, this change is consistent with the rest of the test.